src/garima_s/PatternPrinting.java

```diff
@@ -0,0 +1,55 @@
+/* Write a Program to Print a Following Patterns:
+*
+**
+***
+****
+
+1
+1 2
+1 2 3 
+1 2 3 4
+1 2 3 4 5
+1 2 3 4 5 6
+
+1
+2 3
+4 5 6
+7 8 9 10
+*/
+
+package garima_s;
+
+public class PatternPrinting {
+	void printPatternStar() {
+		for(int i=0;i<=4;i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible